draft-ietf-ppm-dap.md

```diff
@@ -3922,4 +3982,34 @@ Index value:  No transformation needed.
 The initial contents of this namespace are the types and descriptions in
 {{urn-space-errors}}, with the Reference field set to RFC XXXX.
 
+# Extending this Document {#extending-this-doc}
+
+The behavior of DAP may be extended or modified by future documents defining
+one or more of the following:
+
+1. a new batch mode ({{batch-mode}})
+1. a new report extension ({{report-extensions}})
+1. a new report error ({{aggregation-helper-init}})
+1. a new entry in the URN sub-namespace for DAP ({{urn-space-errors}})
+
+Each of these requires registration of a codepoint or other value; see
+{{iana}}. No other considerations are required except in the following cases:
+
+* When a document defines a new batch mode, it MUST include a section titled
+  "DAP Batch Mode Considerations" specifying the following:
```

Comment:
```suggestion
* When a document defines a new batch mode, it MUST include a section specifying the following:
```

This is opinionated, but I don't think we should be so prescriptive as to include the name of the section in the MUST. :)